frontend/public/components/storage-class-form.tsx

```diff
@@ -462,10 +462,10 @@ const StorageClassFormInner: React.FC<StorageClassFormProps> = (props) => {
           <label className={css({ 'co-required': paramIsRequired(key) })} htmlFor={paramId}>
             {_.get(parameter, 'name', key)}
           </label>
-          <Dropdown
+          <ConsoleSelect
```

Comment:
seeing some issue on StorageClass creation form, dropdown value not updated in dropdown

https://github.com/user-attachments/assets/b12a63e9-ffd0-419f-b7c7-4a47822d548f